priv/templates/solaris/Makefile

```diff
@@ -50,7 +50,9 @@ prototype:
 	echo "i r.preserve" >> prototype
 	echo '' >> prototype
 	pkgproto rel/{{package_install_name}}={{package_install_name}} >> prototype
-	sed -e "s/ $(LOGNAME) .*$$/ {{package_install_user}} {{package_install_group}}/" \
+	sed -e "s/ $(LOGNAME) .*$$/ root bin/" \
```

Comment:
I'm trying to figure out exactly what this sed script does/is for, but it's not totally obvious to me. I'm getting a bit nit-picky here, but it would be nice to add a short comment for clarity, so that we don't have to decipher it from scratch if we revisit this code in the future.